src/OpenLoco/Title.cpp

```diff
@@ -83,13 +163,73 @@ namespace OpenLoco::Title
         initialiseViewports();
         sub_4284C8();
         Gui::init();
-        sub_444357();
+        reset();
         Gfx::invalidateScreen();
         resetScreenAge();
 
         Audio::playTitleScreenMusic();
     }
 
+    // 0x00444387
+    void update()
+    {
+        if (!isTitleMode())
+            return;
+
+        resetScreenAge();
+
+        if (_waitCounter > 0)
+        {
+            _waitCounter -= 1;
+            return;
+        }
+
+        do
+        {
+            auto& command = *_sequenceIterator++;
+            std::visit(overloaded{
+                           [](WaitStep step) {
+                               _waitCounter = step.duration / 4;
+                           },
+                           [](ReloadStep step) {
+                               loadTitle();
+                               Gfx::invalidateScreen();
+                               resetScreenAge();
+                               addr<0x50C19A, uint16_t>() = 55000;
+                           },
+                           [](MoveStep step) {
+                               if (addr<0x00525E28, uint32_t>() & 1)
+                               {
+                                   auto pos = Map::map_pos(step.x * 32 + 16, step.y * 32 + 16);
```

Comment:
And now you can use the tile pos constructor to do the *32

Reply:
Which constructor is that? I found one that divides by tile_size (32), but not the other way around.
```
        TilePos(const map_pos& pos)
            : x(pos.x / tile_size)
            , y(pos.y / tile_size)
        {
        }
```
From this, I took TilePos to be the 'smaller' coordinate system, whereas map_pos is the bigger one. Is that correct?

Reply:
My PR #809 solves this all on its own by providing a resolution to the coordinates. Map position is defined with a resolution of 1 and TilePos with a resolution of 32, so when trying to convert them it always knows what to do.